### ISSUE TITLE
Could not pass event PlayerInteractEvent to EnderContainers v2.1.0

### ISSUE THREAD

**Issue by andris155** [utarwyn/EnderContainers#22]
```
[00:28:59] [Server thread/ERROR]: Could not pass event PlayerInteractEvent to EnderContainers v2.1.0
java.lang.IllegalArgumentException: Cannot get ID of Modern Material
	at org.apache.commons.lang.Validate.isTrue(Validate.java:136) ~[patched_1.14.1.jar:git-Paper-23]
	at org.bukkit.Material.getId(Material.java:3319) ~[patched_1.14.1.jar:git-Paper-23]
	at fr.utarwyn.endercontainers.dependency.PlotSquaredDependency.onBlockChestOpened(PlotSquaredDependency.java:41) ~[?:?]
	at fr.utarwyn.endercontainers.dependency.DependenciesManager.onBlockChestOpened(DependenciesManager.java:77) ~[?:?]
	at fr.utarwyn.endercontainers.enderchest.EnderChestListener.onPlayerInteract(EnderChestListener.java:74) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor912.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.14.1.jar:git-Paper-23]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.14.1.jar:git-Paper-23]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.14.1.jar:git-Paper-23]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:534) ~[patched_1.14.1.jar:git-Paper-23]
	at org.bukkit.craftbukkit.v1_14_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:439) ~[patched_1.14.1.jar:git-Paper-23]
	at net.minecraft.server.v1_14_R1.PlayerInteractManager.a(PlayerInteractManager.java:441) ~[patched_1.14.1.jar:git-Paper-23]
	at net.minecraft.server.v1_14_R1.PlayerConnection.a(PlayerConnection.java:1387) ~[patched_1.14.1.jar:git-Paper-23]
	at net.minecraft.server.v1_14_R1.PacketPlayInUseItem.a(PacketPlayInUseItem.java:27) ~[patched_1.14.1.jar:git-Paper-23]
	at net.minecraft.server.v1_14_R1.PacketPlayInUseItem.a(PacketPlayInUseItem.java:5) ~[patched_1.14.1.jar:git-Paper-23]
	at net.minecraft.server.v1_14_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:18) ~[patched_1.14.1.jar:git-Paper-23]
	at net.minecraft.server.v1_14_R1.TickTask.run(SourceFile:18) ~[patched_1.14.1.jar:git-Paper-23]
	at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:127) ~[patched_1.14.1.jar:git-Paper-23]
	at net.minecraft.server.v1_14_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.14.1.jar:git-Paper-23]
	at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:105) ~[patched_1.14.1.jar:git-Paper-23]
	at net.minecraft.server.v1_14_R1.MinecraftServer.aW(MinecraftServer.java:995) ~[patched_1.14.1.jar:git-Paper-23]
	at net.minecraft.server.v1_14_R1.MinecraftServer.executeNext(MinecraftServer.java:988) ~[patched_1.14.1.jar:git-Paper-23]
	at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeAll(IAsyncTaskHandler.java:91) ~[patched_1.14.1.jar:git-Paper-23]
	at net.minecraft.server.v1_14_R1.MinecraftServer.sleepForTick(MinecraftServer.java:971) ~[patched_1.14.1.jar:git-Paper-23]
	at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:906) ~[patched_1.14.1.jar:git-Paper-23]
	at java.lang.Thread.run(Thread.java:834) [?:?]
```

**Comment by utarwyn** [comment]
This bug should be fixed with the commit 31fd57ce6878bd12c5fbc6d2eace77a37272246f.
As always, thank you for the feedback, and sorry for the wait...

Modern materials are not fully supported by the plugin for now, but I have planned to work on it. I hope soon. 👍

**Comment by utarwyn** [comment]
I don't know why I have said "Modern materials are not fully supported" last august. The plugin works from Bukkit 1.8 to 1.15 so all materials should be supported. Anyway, I close this issue as the bug has been fixed a while ago.
